packages/lit-html/src/lit-html.ts

```diff
@@ -310,6 +310,54 @@ const tag =
 /**
  * Interprets a template literal as an HTML template that can efficiently
  * render to and update a container.
+ *
+ * ```ts
+ * const header = (title: string) => html`<h1>${title}</h1>`;
+ * ```
+ *
+ * The `html` tag takes the literal strings and embedded expression values for
```

Comment:
Does this text come from lit.dev, or is it new?

It does seem like there's a downside to having **too detailed** documentation here. Otherwise we are maintaining the same documentation in two different places, which will definitely get out of sync.

I wonder if we should do something a little more concise, and then to the relevant lit.dev section?

Reply:
Yeah, I don't think we need to describe how this works. I think we might hit a couple of points like:
- this returns a description of the DOM to render as a value
- Maybe that the value is a TemplateResult (but even that's pretty drilled in)
- It captures the expression values and the static template strings
- The result needs to be passed to render() to do anything
- If a template result comes from the same expression as a previously rendered result, it's updated instead of replaced
- Point to the template docs on lit.dev

Reply:
Agreed! Do we have this info on lit.dev (I couldn't find it). My reference was: https://lit-html.polymer-project.org/guide/writing-templates

Is the plan that lit.dev contains all the documentation for various packages?

Reply:
Yes, lit.dev is completely replacing the other sites.

Reply:
I opened issue: https://github.com/lit/lit.dev/issues/504, since not sure if we have a lit.dev location for these inner concepts yet.

Made much more succinct in follow up commit. Thank you!